### PR TITLE
[250] Header Menu content pages ([...] Menu)

### PR DESCRIPTION
**Pull request by W3stside**
Relevant to #418 and to #250 

@anxolin I'm opening this as it was related to the Footer and the content pages + to see how similar our PRs are based on your slack message that you also are working on adding this back in

1. added the content pages (essentially the footer links) into the menu 
2. added icons to the menu items (just picked random ones can be re-explored)
3. created InternalMenuItem for internal links

### Desktop
Screenshot from 2021-04-14 13-04-04

### Mobile
Screenshot from 2021-04-14 13-04-51

## QA
1. Just open the menu and check that the links all work and go to right places
2. for now the About page is not linking anywhere proper as we're waiting #308

### CODE REVIEW

**Comment by biocom** [comment]
Looks good. One point: There's interference with the feedback button on both mobile and desktop (good reason to have this Feedback button for testing purposes in this case!):

Screen Shot 2021-04-14 at 15 29 40
Screen Shot 2021-04-14 at 15 29 29


My suggestions:
- Offset the position of the feedback button to move it down on desktop and up on mobile.
- Start a z-index war.

**Review by anxolin — APPROVED** [review]
Nice! Looks good. I've made some suggestions but they came in a PR on top of this branch

**Review by biocom — APPROVED** [review]

File: src/custom/components/Menu/index.tsx [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 import styled from 'styled-components'
+import { FOOTER_URLS } from '../Footer'
 
 const CODE_LINK = 'https://github.com/gnosis/dex-swap'
 const DISCORD_LINK = 'https://chat.gnosis.io'
 
 export const StyledMenu = styled(MenuMod)``
 
+// remove static gnosis text and leave urls
+const CONTENT_URLS = FOOTER_URLS.slice(1) as { name: string; url: string; Icon?: React.FC<IconProps> }[]
```
Comment (alfetopito) [inline src/custom/components/Menu/index.tsx:14]: Why not remove it from the `FOOTER_URLS` then?

**Review by alfetopito — APPROVED** [review]

File: src/custom/components/Menu/index.tsx (hunk 1 above)
Comment (anxolin) [reply src/custom/components/Menu/index.tsx:14]: yep, i think we are removing the footer links. They will dissapear when we consolidate things.

**Pull request merged**